stabilizer/src/main/java/com/hazelcast/stabilizer/worker/selector/OperationSelectorBuilder.java

```diff
@@ -0,0 +1,86 @@
+package com.hazelcast.stabilizer.worker.selector;
+
+import org.apache.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.format;
+
+/**
+ * Builder to create an {@link OperationSelector} instance. Each operations is represented as an enum item
+ * and is registered with its probability percentage (0 to 100) via the {@link #addOperation(Enum, int)} method.
+ * <p/>
+ *
+ * @param <T> enum of operations
+ */
+public class OperationSelectorBuilder<T extends Enum<T>> {
```

Comment:
Apart from that; every worker will get its own OperationSelector so therefore it doesn't need to be threadsafe. That is what you need to put in the documentation so that people know how to use the API correctly